src/Datadog.Trace/Vendors/Serilog/Parsing/PropertyToken.cs

```diff
@@ -32,7 +32,10 @@ namespace Datadog.Trace.Vendors.Serilog.Parsing
     internal sealed class PropertyToken : MessageTemplateToken
     {
         readonly string _rawText;
-        readonly int? _position;
+        // Remove readonly keyword on field. This leads to code that may be evaluated as unverifiable on .NET Framework
```

Comment:
This might be the first change we've made to the vendored Serilog (besides namespaces and access modifiers). Are we at a point where we should break out a new fork of the original project for this?